mysqlnd_azure.c

```diff
@@ -654,13 +689,15 @@ MYSQLND_METHOD(mysqlnd_azure, connect)(MYSQLND * conn_handle,
 
                     ret = (*pconn)->m->connect(pconn, redirect_host, redirect_user, password, database, redirect_info->redirect_port, socket_or_pipe, mysql_flags);
                     if (ret == FAIL) {
+                        AZURE_LOG(ALOG_LEVEL_INFO, "Redirect cache miss");
                         //remove invalid cache
                         mysqlnd_azure_remove_redirect_cache(username.s, hostname.s, port);
 
                         ret = (*pconn)->m->connect(pconn, hostname, username, password, database, port, socket_or_pipe, mysql_flags);
                     }
```

Comment:
if ret = PASS, it will be a very useful log if we can add "Using cache to connect succeeded"